js/render-big-picture.js

```diff
@@ -4,16 +4,17 @@ const bigPicture = document.querySelector('.big-picture');
 const countLikes = bigPicture.querySelector('.likes-count');
 const photoImage = bigPicture.querySelector('.big-picture__img img');
 const photoTitle = bigPicture.querySelector('.social__caption');
+const loaderButton = document.querySelector('.comments-loader');
+const commentCount = document.querySelector('.social__comment-count');
 
+const preparationComments = (min, max, comments) => {
```

Comment:
в функциях лучше глаголы использовать
```suggestion
const preparateComments = (min, max, comments) => {
```
также обычная практика нейминга таких функций преобразователей format + ...